examples/guides/se-restful-webservice/README.adoc

```diff
@@ -642,31 +547,31 @@ These link the health-related endpoints your code is exposing to the new methods
 
 // tag::tryReadiness[]
 
-=== Try the readiness check
-Access the readiness check endpoint:
+=== Check the server's health
+Run this command
 [source,bash]
-curl -i -X GET http://localhost:8080/ready
+curl -i -X GET http://localhost:8080/health
 
-Our readiness check returns no payload, just the 200 status, so you won't see any data
-displayed. The `-i` option shows the
-200 status in the response.
-// end::tryReadiness[]
-
-// tag::tryLiveness[]
+and you should see output like this
```

Comment:
You should see output as shown in this example:

Reply:
fixed